apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTimings.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Storage.V1;
+
+/// <summary>
+/// These specify the custom timing configurations for retrying in case of failure of the API request.
+/// </summary>
+public sealed class RetryTimings
+{
+    /// <summary>
+    /// The backoff time between the first attempt and the first retry. It must be non-negative. 
+    /// </summary>
+    public TimeSpan InitialBackoff { get; private set; }
```

Comment:
I'd prefer to make all these properties *properly* read-only. Instead, we should always use the parameterized constructor accepting all values. That makes it easier to tell that it's really immutable.